### ISSUE TITLE
Can we add a Command set to Site Pages command bar?

### ISSUE THREAD

**Issue by ashishshukla1183** [SharePoint/sp-dev-fx-extensions#181]
## Category

- [X] Question
- [ ] Bug
- [ ] Enhancement

Is there any way to add actions/commands to Site pages command bar using Command sets?
Please refer the attached image for the exact location.

![image](https://user-images.githubusercontent.com/10719499/48874377-938c2200-edb8-11e8-9087-579ab27e4122.png)

**Comment by luismanez** [comment]
I don´t think you can do that. CommandSets are only tied to Lists (I think that would be supported at some point, but just an opinion). A non-recommended approach could be to use an Application Extension, and deal with the DOM to place the button in there, but as I say, is not recommended by MS...

**Comment by ashishshukla1183** [comment]
> 
> 
> I don´t think you can do that. CommandSets are only tied to Lists (I think that would be supported at some point, but just an opinion). A non-recommended approach could be to use an Application Extension, and deal with the DOM to place the button in there, but as I say, is not recommended by MS...

That is an option but not sure how long as one update can break everything. It would be great if command set is not just limited to listview pages.

**Comment by hugoabernier** [comment]
Ashish, I created a uservoice entry for this a while ago, but it isn't getting a lot of votes. Feel free to vote:
https://sharepoint.uservoice.com/forums/329220-sharepoint-dev-platform/suggestions/33029431-modern-page-extensions

Also, the reason why it doesn't get votes is probably because my description isn't quite clear. Feel free to add to it.
